sema4ai/src/sema4ai_code/agent_cli.py

```diff
@@ -1,19 +1,23 @@
 import os
 import weakref
-from typing import Optional, Any
+from pathlib import Path
+from typing import Any, Optional
 
+from sema4ai_ls_core import uris
 from sema4ai_ls_core.core_log import get_logger
-from sema4ai_code.tools import Tool
-
+from sema4ai_ls_core.jsonrpc.monitor import IMonitor
 from sema4ai_ls_core.protocols import (
+    ActionResultDict,
     IConfig,
     IConfigProvider,
+    IWorkspace,
 )
 
-from sema4ai_code.protocols import (
-    ActionResult,
-    AgentCliResult,
+from sema4ai_code.agents.collect_agent_spec_diagnostics import (
```

Comment:
Make this import local (for really common things imports can be global, but otherwise, try to default to local imports (so that we don't pay the import time price at startup).